test/integration/targets/nxos_snmp_contact/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- set_fact: connection="{{ cli }}"
+
+- import_tasks: "{{ role_path }}/tests/common/sanity.yaml"
```

Comment:
This file is not necessary as the logic is handled in tasks/. This will run cli tests twice for both connections.